linkedlist.c

```diff
@@ -0,0 +1,64 @@
+/* Drew French */
+
+/* Does stuff with
+ * linked lists.
+ */
+
+/* Did not finish; trying
+ * to figure out why what
+ * there is so far is
+ * getting errors.
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct node
+{
+	int n;
+	struct node *next;
+} nodeType;
+
+void printList(nodeType *head)
+{
+	nodeType *current = head;
+	while(current != NULL)
+	{
+		printf("%d\n", current->n)
+		current = current->next;
+	}
+}
+
+void clear(nodeType *head)
+{
+	nodeType *current = head;
+	while(current != NULL)
+	{
+		free(*current);
+	}
+}
+
+int main()
+{	
+	nodeType *head = NULL;
+
+	head = malloc(sizeof(nodeType));
+	head->n = 3;
+	head->next = malloc(sizeof(nodeType));
+	head->next->n = 1;
```

Comment:
these steps should have been done through the `insert()` function! see how you have to rewrite the code each time? That's why we use functions!